doc/source/Resources/pyaedt_installer_from_aedt.py

```diff
@@ -39,6 +39,16 @@
 
 
 VENV_DIR_PREFIX = ".pyaedt_env"
+
+VENV_DIR = None  # "e:/pyaedt_env"
```

Comment:
Please remove the comment and add a brief explanation in the previous line (as a comment), to explain that the user can define a specific virtual environment.